api/src/main/java/javax/enterprise/context/Destroyed.java

```diff
@@ -33,13 +33,12 @@
 
 /**
  * <p>
- * The <code>@Destroyed</code> qualifier.
+ * This qualifier is deprecated from CDI 2.0 and {@link BeforeDestroyed} or {@link AfterDestroyed} should be used instead.
```

Comment:
Shouldn't we add a note that the new `@AfterDestroyed` behaves equally to this deprecated one?